dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Response } from 'express';
+import { CustomExpressRequest } from '../../../types/custom-express-request';
+import { getUtilisationReportDetailsByBankId, getUtilisationReportDetailsByBankIdMonthAndYear } from '../../../services/repositories/utilisation-reports-repo';
+
+type GetUtilisationReportsRequest = CustomExpressRequest<{
+  params: {
+    bankId: string;
+  };
+  query: {
+    month: string | undefined;
+    year: string | undefined;
+  };
+}>;
+
+export const getUtilisationReports = async (req: GetUtilisationReportsRequest, res: Response) => {
```

Comment:
Missing docstring, please ensure all new functions have docstrings.

Reply:
Do we need to add documentation for typescript functions like these? In this case, the naming and typing of parameters/return values means that adding code does not provide a benefit and adds a small maintenance cost. As we're aiming to write self-documenting code I think additional code would go against DRY principles

Reply:
Docstrings provide documentation for the code, making it easier for others to understand the purpose, usage, and expected behaviour of functions. Additionally, many development tools and IDEs use these comments to provide helpful information and suggestions while coding.

I use this for reference https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html.